src/language/__tests__/blockString-fuzz.js

```diff
@@ -66,3 +70,77 @@ describe('printBlockString', () => {
     }
   }).timeout(20000);
 });
+
+describe('isPrintableBlockString', () => {
+  it('correctly tells if random strings are printable block string', () => {
+    // Testing with length >7 is taking exponentially more time. However it is
+    // highly recommended to test with increased limit if you make any change.
+    for (const fuzzStr of genFuzzStrings({
+      allowedChars: ['\n', '\t', ' ', '"', 'a', '\\'],
+      maxLength: 7,
+    })) {
+      if (isPrintableBlockString(fuzzStr)) {
+        const testStr = '"""' + fuzzStr + '"""';
+
+        let testValue;
+        try {
+          testValue = lexValue(testStr);
+        } catch (e) {
+          continue; // skip invalid values
+        }
+        invariant(typeof testValue === 'string');
+
+        const printedValue = lexValue(printBlockString(testValue));
+
+        invariant(
+          testValue === printedValue,
```

Comment:
It should be:
```js
lexValue(printBlockString(fuzzStr)) === fuzzStr
```

Reply:
and you can remove all previous code

Reply:
fuzzStr can have lines with only whitespaces or trailing or leading multiple new lines which will be removed on print. Can't compare the 2.

Reply:
Yes, this is exactly the purpose of `isPrintableBlockString` to detect such strings.
Any string can be printed as standard strings but only subset of strings can be safely printed as block strings so `isPrintableBlockString` should be predicate function to detect them.